src/extension.ts

```diff
@@ -75,6 +79,10 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(runCommand);
     context.subscriptions.push(solidityWatcher);
+
+    item.text = "$(beaker) Aderyn Start";
```

Comment:
In debug mode, the extension doesn't seem to highlight the code diagnostics. I'm not sure why.